data/json/overmap/overmap_special/specials.json

```diff
@@ -4547,6 +4547,36 @@
     "rotate": false,
     "flags": [ "UNIQUE", "SAFE_AT_WORLDGEN" ]
   },
+  {
+    "type": "overmap_special",
+    "id": "isolated_road",
+    "overmaps": [
+      { "point": [ 0, 0, 0 ], "overmap": "isolated_road_field_0" },
+      { "point": [ 1, 0, 0 ], "overmap": "isolated_house_farm_gunsmith" },
+      { "point": [ 0, 1, 0 ], "overmap": "isolated_house_farm_blacksmith" },
+      { "point": [ 1, 0, 1 ], "overmap": "isolated_house_farm_roof_gunsmith" },
+      { "point": [ 0, 1, 1 ], "overmap": "isolated_house_farm_roof_blacksmith" },
+      { "point": [ 1, 1, 0 ], "overmap": "isolated_road_field_1" }
+    ],
+    "connections": [ { "point": [ 0, -1, 0 ], "terrain": "road", "connection": "local_road", "from": [ 0, 0, 0 ] } ],
+    "locations": [ "field" ],
+    "city_distance": [ 3, -1 ],
+    "city_sizes": [ 1, 16 ],
+    "occurrences": [ 33, 100 ],
```

Comment:
33% chance to spawn per overmap feels way too much for this location. It will spawn too frequently despite supposed to be unique.

Reply:
I copied that from Hub 01 since I wasn't exactly sure what that value should be. 

I find even with the 8% for the EXODII base a lot of times the quests to reveal the location will fail to find any instance of it returning a failure. 

If it is permissible can it remain high for now and then be dropped after some feedback similar to the TCL and others?

Reply:
Sure, leave it as is until feedback will prove it spawns really too frequently.

Reply:
To be clear I meant feedback on the content like how the TCL has an increased spawn rate currently and will be reduced before .G. I didn't realize it was 33% that's definitely too high.